application/ui/components/api-resource.jsx

```diff
@@ -51,13 +51,15 @@ module.exports = React.createClass({
 
         if (resource) {
             window.document.title = resource.title;
+
             resourceComponent = (
                 <Resource
-                    name     = {resource.name}
-                    synopsis = {resource.synopsis}
-                    methods  = {resource.methods}
+                    name       = {resource.name}
+                    synopsis   = {resource.synopsis}
+                    endpoints  = {_.isUndefined(resource.endpoints) ? resource.methods : resource.endpoints}
```

Comment:
That sounds good... Can we also add a `v1.0` issue to this repo to deprecate `methods`?